templates/default/inspec_default_test.rb.erb

```diff
@@ -1,4 +1,4 @@
 # InSpec test for recipe <%= cookbook_name %>::<%= recipe_name %>
 
 # The InSpec reference, with examples and extensive documentation, can be
-# found at http://inspec.io/docs/reference/resources/
+# found at https://www.inspec.io/docs/reference/resources/
```

Comment:
Inspec docs have moved:
```suggestion
# found at https://docs.chef.io/inspec/resources/
```

Reply:
You should probably make an update in the the upstream chef-cli repo then as I copied this directly from there :laughing: